<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2LNVLS7" ignored="0" name="Test" tests="3" failures="1" timestamp="20 Aug 2022 10:33:23 GMT" time="3.338" errors="0">
  <testcase name="test01" time="2.021" classname="com.tests.LoginTest"/>
  <testcase name="test02" time="0.01" classname="com.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at com.tests.LoginTest.test02(LoginTest.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
  <testcase name="test03" time="0.007" classname="com.tests.LoginTest">
    <skipped/>
  </testcase> <!-- test03 -->
</testsuite> <!-- Test -->
